TeachEasy/src/teacheasy/data/AnswerBoxData.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Sam Raeburn
+ * Sam Hall
+ * Copyright (c) 2015 Sofia Software Solutions. All Rights Reserved.
+ */
+package teacheasy.data;
+
+/**
+ * Class for holding answer box data
+ * @author sr896 and sh1157
+ *
+ */
+public class AnswerBoxData extends PageObject {
+	int characterLimit, marks;
+	String correctAnswers;
+	boolean retry;

```

Comment:
Missing access specifiers.

Style: Variables should be declared on new lines unless closely related (x and y)